src/PointwiseFunctions/AnalyticSolutions/Elasticity/BentBeam.hpp

```diff
@@ -112,6 +113,9 @@ class BentBeam {
     return constitutive_relation_;
   }
 
+  /// Return potential energy integrated over the whole domain
```

Comment:
perhaps replace `domain` with `beam material` and/or make clear that the integral goes over the entire height and length of the beam, independent of what the computational domain might be. It's probably best to write down the exact expression that is being computed here, i.e. U=\int...=6 l M^2...

Reply:
Also seeing how this isn't marked outdated, I realize I confused BentBeam.hpp and Test_BentBeam.cpp. I will address that later as well.